pep-0517.txt

```diff
@@ -503,6 +540,48 @@ package *authors*, while still allowing *end-users* to open up the
 hood and apply duct tape when necessary.
 
 
+In-Tree Build Backends
+======================
+
+In certain circumstances, projects may wish to include the source code for the
+build backend directly in the source tree, rather than referencing the backend
+via the ``requires`` key. Two specific situations where this would be expected
+are:
+
+- Backends themselves, which want to use their own features for building
+  themselves ("self-hosting backends")
+- Project-specific backends, typically consisting of a custom wrapper around a
+  standard backend, where the wrapper is too project-specific to be worth
+  distributing independently ("in-tree backends")
+
+Projects can specify that their backend code is hosted in-tree by including the
+``python-paths`` key in ``pyprject.toml``. This key contains a list of
```

Comment:
```suggestion
``python-paths`` key in ``pyproject.toml``. This key contains a list of
```

Reply:
Bah, I made that correction (and the one you note below where I'm inconsistent between `python-path` and `python-paths`) in my local copy, but it looks like I didn't update my github copy before creating the PR. I'll fix these.